frontend/fingertips-frontend/lib/registry.test.tsx

```diff
@@ -32,7 +32,8 @@ describe('StyledComponentsRegistry', () => {
     expect(container.asFragment()).toMatchSnapshot();
   });
 
-  it('should not return inline styles when rendering the wrapped component server side', () => {
+  it.skip('should not return inline styles when rendering the wrapped component server side', () => {
```

Comment:
Yeah this test is an issue. I'm happy for you to delete this test file. Discussed with @williamwillis11 and we think this test will be better placed as an e2e playwright test

Reply:
deleted the test files